src/renderer/vt/VtSequences.cpp

```diff
@@ -445,3 +446,16 @@ using namespace Microsoft::Console::Render;
 {
     return _Write("\x1b[?9001h");
 }
+
+// Method Description:
+// - Formats and writes a sequence to add a hyperlink to the terminal buffer
+// Arguments:
+// - The hyperlink URI
+// Return Value:
+// - S_OK if we succeeded, else an appropriate HRESULT for failing to allocate or write.
+[[nodiscard]] HRESULT VtEngine::_SetHyperlink(std::wstring uri) noexcept
+{
+    const auto convertedUri = ConvertToA(CP_UTF8, uri);

```

Comment:
We should use `til::u16u8` for this -- it's a one-shot conversion from utf16 to utf8 that doesn't require any codepage magic :smile:

Reply:
Fixed!